source/_includes/common-tasks/backups.md

```diff
@@ -12,6 +12,16 @@ A full backup includes the following directories:
 
 A partial backup consists of any number of the above default directories and installed add-ons.
 
+### Preparing for a backup
+
+1. Before creating a backup, check if you can reduce the size of the backup.
+   - Check if your configuration directory contains a large database file (`home-assistant_v2.db`). 
+   - See the [`recorder`](/integrations/recorder/) integration page for options to keep your database data down to a size that won't cause issues. 
```

Comment:
This makes it seem like everyone should do this.
By default, it only keeps 10 days, which should not cause any problems for regular users.